src/components/10-atoms/input-text/index.js

```diff
@@ -142,73 +142,71 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
     };
 
     return html`
-      <div class="a-input-text__wrapper">
-        ${label &&
-          html`
-            <label for="${refId}" class="a-input-text__label">
-              ${label}
-              ${required
-                ? html`
-                    *
-                  `
-                : ''}
-            </label>
-          `}
-        <div class="a-input-text__input-wrapper">
-          <div class="a-input-text__input-elements">
-            <input
-              id="${refId}"
-              type="${type}"
-              class="${classMap(inputClasses)}"
-              autocomplete="off"
-              name="${name}"
-              value="${value}"
-              placeholder="${placeholder}"
-              aria-required="${required}"
-              ?disabled="${disabled}"
-              @input="${this.handleInput}"
-              @focus="${this.handleFocus}"
-              @blur="${this.handleBlur}"
-            />
-            ${
-              checkMark
-                ? html`
-                    <span class="a-input-text__check"></span>
-                  `
-                : ''
-            }
-          </div>
-          ${info &&
-            html`
-              <axa-popup-button
-                ?open="${this._open}"
-                class="a-input-text__info-button"
-                @click="${this.handlePopupButtonClick}"
-              ></axa-popup-button>
-            `}
-        </div>
+      ${label &&
+        html`
+          <label for="${refId}" class="a-input-text__label">
+            ${label}
+            ${required
+              ? html`
+                  *
+                `
+              : ''}
+          </label>
+        `}
+      <div class="a-input-text__input-wrapper">
+        <div class="a-input-text__input-elements">
+          <input
+            id="${refId}"
+            type="${type}"
+            class="${classMap(inputClasses)}"
+            autocomplete="off"
+            name="${name}"
+            value="${value}"
+            placeholder="${placeholder}"
+            aria-required="${required}"
+            ?disabled="${disabled}"
+            @input="${this.handleInput}"
+            @focus="${this.handleFocus}"
+            @blur="${this.handleBlur}"
+          />
           ${
-            this.showError
+            checkMark
               ? html`
-                  <span class="a-input-text__error">${error}</span>
+                  <span class="a-input-text__check"></span>
                 `
               : ''
           }
         </div>
+        ${info &&
+          html`
+            <axa-popup-button
+              ?open="${this._open}"
+              class="a-input-text__info-button"
+              @click="${this.handlePopupButtonClick}"
+            ></axa-popup-button>
+          `}
+      </div>
         ${
-          info
+          this.showError
```

Comment:
Why not include showError in initial destructuring of this?

Reply:
I don't destruct functions, but yeah it's not that obvious.